### PR TITLE
Feat/use new schema changing queries

### PR DESCRIPTION
**Pull request by hellypi**
starts with Fix: revert package* changes

### CODE REVIEW

**Comment by hellypi** [comment]
[Make l-s use the new schema-changing queries](https://trello.com/c/fp12IX8Z/605-make-l-s-use-the-new-schema-changing-queries)

**Review by hellypi — COMMENTED** [review]
.

File: lib/compute-schema.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const computeSchema = (req, res, next) => {
+  const store = new Map()
+  store.set('schema', 'public')
+  store.set('defaultSchema', 'ERROR_SCHEMA')
+  asyncLocalStorage.run(store, () => {
+    next()
+  })
+}
+
+module.exports = computeSchema
```
Comment (josepfo) [inline lib/compute-schema.js]: Seems like the first 3 lines of this function could be replaced by the defaultStore() from ./async-storage. What do you think about using it to create the store here?

File: lib/db.js [hunk 2]
```diff
@@ -23,13 +48,17 @@ eventBus.subscribe('log', args => {
...
-  const msgToSave = message ? message : _.get('message', meta)
+  const msgToSave = message || _.get('message', meta)
 
   const sql = `insert into server_logs
   (id, device_id, message, log_level, meta) values ($1, $2, $3, $4, $5) returning *`
-
-  db.one(sql, [uuid.v4(), '', msgToSave, level, meta])
-    .then(_.mapKeys(_.camelCase))
+  // need to set AsyncLocalStorage (ALS) for this function as well
+  // because this module is imported before ALS is set up on app.js
```
Comment (josepfo) [inline lib/db.js:56]: Are these still useful to understand the code?

**Review by josepfo — COMMENTED** [review]
Left you some comments. But looks good.

File: lib/compute-schema.js (hunk 1 above)
Comment (hellypi) [reply lib/compute-schema.js]: Since this is temporary(?) hard-coded stuff I didn't want to use defaultStore() but I think you're right

**Review by josepfo — APPROVED** [review]

**Pull request closed without merging**